index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="fill-100">
+    <nav class="navigation">
+      <span>
+        <a href="https://">
+          <img class="icon" src="assets/images/Smile.svg" alt="Javi smiling icon">
+        </a>
+      </span>
+      <span>
+        <a href="http://">
+          <img class="icon" src="assets/images/menu.svg" alt="navigation menu">
+        </a>
+      </span>
+    </nav>
+  </header>
+  <main class="fill-100 pv5">
+    <article>
+      <section>
+        <h1>I'm Javi,<br>
+          Web Designer<br>
+          that codes</h1>
+        <p class="pv2">I make ads and websites, combined with code to improve the conversion rates of their marketing</p>
+        <a href="https://ly.enddesign.co/intro">Let's connect</a>
+        <div class="pv5">
+          <span>
+            <a href="http://">
+              <img class="icon" src="assets/images/linkedin.svg" alt="Linkedin profile @javierjhm">
+            </a>
+          </span>
+          <span>
+            <a href="http://">
+              <img class="icon" src="assets/images/github.svg" alt="Github profile @wickathou">
+            </a>
+          </span>
+          <span>
+            <a href="http://">
+              <img class="icon" src="assets/images/instagram.svg" alt="Linkedin profile @digijavi">
+            </a>
+          </span>
+        </div>
```

Comment:
- [OPTIONAL] Please make sure that your social media links are displayed using list items. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
  - for example
  ```css
  <ul>
   <li>
     <a><img src="./images/Linkedin icon.png" alt="none"></a>
   </li>
  </ul>
  ```